src/register/register.ts

```diff
@@ -30,9 +32,20 @@ export class Register {
    */
   private static registers: { [key: string]: IRegisterContent } = {
     '"': { text: "", registerMode: RegisterMode.CharacterWise },
-    '*': { text: "", registerMode: RegisterMode.CharacterWise }
+    '*': { text: "", registerMode: RegisterMode.CharacterWise, isClipboardRegister: true },
+    '+': { text: "", registerMode: RegisterMode.CharacterWise, isClipboardRegister: true }
   };
 
+  public static isClipboardRegister(registerName: string): boolean {
+    const register = Register.registers[registerName];
+
+    if (register && register.isClipboardRegister) {
```

Comment:
```typescript
  public static isClipboardRegister(registerName: string): boolean {
    const register = Register.registers[registerName];
    return register && register.isClipboardRegister;
  }
```

![capture](https://cloud.githubusercontent.com/assets/1448597/18657782/71d35128-7ecb-11e6-9f65-b8dd3abdb263.PNG)

I think it's being caused by the `isClipboardRegister` property being optional: the property has a type signature of `boolean | undefined`, but the `Register.isClipboardRegister` method expects to only return booleans. 

If we try to return the property without first converting it to a `boolean`, it's possible for the value to `undefined` (b/c it's optional), which violates the signature for the checking method.

(I think) this would also explain why it works if I change the allowed return type to `boolean | undefined`.

I'm pretty new to typescript, so I'm not sure what the idiomatic way to handle this kind of thing is 😕